students/alheadstrong/lesson_6/test_mailroom_part4.py

```diff
@@ -0,0 +1,91 @@
+from mailroom_part4 import *
+
+
+def test_1():
+    assert is_name_list('not_list') is False
+
+
+def test_2():
```

Comment:
your test names should have meaningful names that are easy for the reader to know what you're testing.

for example:
```
def is_name_list_not_list
def is_name_list_a_list
def is_name_list_empty_string
```